package.json

```diff
@@ -6,7 +6,7 @@
     "medical",
     "cornerstone"
   ],
-  "version": "2.1.5",
+  "version": "2.1.5-fix",
```

Comment:
La versione con -fix non esiste, deve essere sempre incrementale rispetto all'ultima versione in master. Fare un rebase e controllare. (E' da cambiare anche nel README)